frontend/components/surplus/SurplusText.vue

```diff
@@ -3,43 +3,52 @@
         <TextBlock v-if="isExplanationsShown" title="What is the Maker Protocol?" class="TextBlock">
             The Maker Protocol is
             <Explain text="a set of rules">
-                a <a href="https://changelog.makerdao.com/" target="_blank"> set of smart contracts </a> running on the
-                Ethereum blockchain</Explain
-            >
+                a <a href="https://changelog.makerdao.com/" target="_blank">set of smart contracts</a> running on the
+                Ethereum blockchain
+            </Explain>
             that defines how a cryptocurrency called DAI is kept approximately equal to USD by incentivizing market
             players. People who help to keep DAI stable, benefit from their actions by acquiring cryptocurrency at a
             discount. The main promise of the protocol is to provide a decentralized
             <Explain text="stable currency">
-                <a href="https://makerdao.com/en/whitepaper#the-dai-stablecoin" target="_blank">
-                    The DAI Stablecoin
-                </a>
+                <a href="https://makerdao.com/en/whitepaper#the-dai-stablecoin" target="_blank">The DAI Stablecoin</a>
                 is a decentralized, unbiased, collateral-backed cryptocurrency soft-pegged to the US Dollar </Explain
             >, which can be used to borrow money over a longer period without being affected by unpredictable exchange
             rates.
         </TextBlock>
         <TextBlock v-if="isExplanationsShown" title="What are surplus auctions?" class="TextBlock">
-            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore
-            et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
-            Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit
-            amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna
-            aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd
-            gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur
-            sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
-            voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea
-            takimata sanctus est Lorem ipsum dolor sit amet.
+            A
+            <a
+                href="https://docs.makerdao.com/smart-contract-modules/system-stabilizer-module/flap-detailed-documentation"
+                target="_blank"
+            >
+                surplus auction
+            </a>
```

Comment:
Let's not have links directly in the text, but only inside `Explain` texts. I would suggest to wrap `surplus auction` into `Explain` with the text: `In Maker terms surplus auctions are called [flap auctions](https://docs.makerdao.com/smart-contract-modules/system-stabilizer-module/flap-detailed-documentation) and are operated via the [flap.sol contract](https://github.com/makerdao/dss/blob/master/src/flap.sol)`

@LukSteib do you agree?

Reply:
Agreed